src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioServiceImpl.java

```diff
@@ -105,6 +109,20 @@ public InputStream preview(Integer mediaId) {
         return mediaBinaryData.toStream();
     }
 
+    @Override
+    public Flux<ResponseEntity<byte[]>> getAudioPreviewFlux(Integer mediaId, String range) {
+        return Flux.just(mediaId)
+            .publishOn(Schedulers.boundedElastic()).map(m -> {
+                InputStream audioMediaFile = preview(m);
+                try {
```

Comment:
put the input stream inside the try block to get the IS close for free

Reply:
ACCEPT